pkg/resources/job.go

```diff
@@ -133,6 +133,11 @@ var jobSchema = map[string]*schema.Schema{
 		Optional:    true,
 		Description: "Job identifier that this job defers to",
 	},
+	"self_deferring": &schema.Schema{
+		Type:        schema.TypeBool,
+		Optional:    true,
+		Description: "Whether this job defers on a previous run of itself",
```

Comment:
I think there's an arg to the effect of `ConflictsWith`, could you add that to conflict with the deferring_job_id plz?